scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -123,6 +123,26 @@ trait ClassLikeSupport:
           .map { _ =>
             parseMethod(dd.symbol, kind = Kind.Given(getGivenInstance(dd).map(_.asSignature), None))
           }
+          
+      case dd: DefDef if !dd.symbol.isHiddenByVisibility && dd.symbol.isExported =>
+        val exportedTarget = dd.rhs.flatMap { 
+          case a: Apply => Some(a.fun)
+          case s: Select => Some(s)
+          case _ => None
+        }.map { 
+          case s: Select => s
+        } 
+        val functionName = exportedTarget.fold("instance")(_.name)
+        val instanceName = exportedTarget.fold("function")(_.qualifier.asInstanceOf[Select].name)
+        val dri = dd.rhs.flatMap {
```

Comment:
Can `rhs` be empy? If not we can simplify this code a lot

Reply:
Not sure, but I followed the API that returns an `Option`